apps/analysis_framework/serializers.py

```diff
@@ -170,6 +172,9 @@ class AnalysisFrameworkSerializer(RemoveNullFieldsMixin,
     )
 
     is_admin = serializers.SerializerMethodField()
+    can_add_users = serializers.SerializerMethodField()
+    projects = serializers.SerializerMethodField()
```

Comment:
The names `projects` and `projects_count` refer to different sets of projects. Good to have a more descriptive name, something like private projects or all_projects_count.

Reply:
@bewakes Does `all_projects_count` and `visible_projects` sounds good?

Reply:
Yes, sounds perfect!!

Reply:
Changes pushed https://github.com/the-deep/server/pull/470/commits/643fe281782d0cec95d2c7b114ed1734bd22be78